java/piranha/src/main/java/com/uber/piranha/XPFlagCleaner.java

```diff
@@ -820,6 +852,14 @@ public Description matchMethodInvocation(MethodInvocationTree tree, VisitorState
     }
 
     Value x = evalExpr(tree, state);
+    if (x == Value.TRUE || x == Value.FALSE) {
+      if (enclosingNode(MOCKITO_UNNECESSARY_MOCKING_PATTERN).matches(tree, state)) {
+        StatementTree stmt = ASTHelpers.findEnclosingNode(state.getPath(), StatementTree.class);
+        if (stmt != null) {
+          return buildDescription(stmt).addFix(SuggestedFix.delete(stmt)).build();
```

Comment:
can the changes happen in updateCode? If not, maybe update the `startPos` and `endPos` here?